packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -831,6 +834,18 @@ class DefaultMaterialLocalizations implements MaterialLocalizations {
   @override
   String get licensesPageTitle => 'Licenses';
 
+  @override
+  String licensesPackageDetailText(int licenseCount) {
+    switch (licenseCount) {
+      case 0:
```

Comment:
If you skip packages with zero licenses, this case should assert.

Reply:
Added assertion licenseCount >= 1, removed case 0.